test/alda/parser/duration_test.clj

```diff
@@ -31,14 +39,20 @@
               (alda.lisp/duration (alda.lisp/ms 2000)))]))))
 
 (deftest tie-and-slur-tests
-  (testing "ties"
+  (testing "ties and slurs"
```

Comment:
In retrospect, I'm not sure why I had it nested like this in the first place:

```clojure
  (testing "ties"
    (testing "ties"
      ...
```

Probably a silly error my part. I think we can remove a layer of nesting and just do this:

```clojure
(deftest tie-and-slur-tests
  (testing "ties"
    ...)
  (testing "slurs"
    ...))
```